src/app/visit/visit.component.ts

```diff
@@ -73,23 +72,31 @@ export class VisitComponent implements OnInit {
         const index = this.visitPoints.indexOf(row.id);
         if (index !== -1) {
             this.visitPoints.splice(index, 1);
+            this.showCheck = false;
         } else {
             this.visitPoints.push(row.id);
         }
+        if (!this.paginator.hasNextPage() && this.visitPoints.length >= 1) {
+            this.showCheck = true;
+        }
```

Comment:
this block of code is repeated 3 times, it is a good moment to move it to a private function and call it 3 times the function

Reply:
done